src/includes/docs-nav.html

```diff
@@ -45,6 +45,7 @@
 		<li><a href="/docs/conventions">Conventions</a></li>
 		<li><a href="/docs/config-adapters">Config Adapters</a></li>
 		<li><a href="/docs/logging">How Logging Works</a></li>
+		<li><a href="/docs/metrics">Monitoring Caddy</a></li>
```

Comment:
I think "Monitoring" means something else for most people; maybe we should simply change this to "Metrics"?

Reply:
Maybe... 🤔 The metrics are there _for_ monitoring Caddy though... What else would you think of when you read "monitoring?"

Reply:
I usually associate Monitoring with Alerting -- of course these are separate things, but I dunno if others have the same associations. What about "Exporting Metrics"?

Or we can leave it. It's not a big deal. :man_shrugging:

Reply:
Yup, this is _also_ for alerting. Prometheus handles alerting - see https://prometheus.io/docs/alerting/latest/overview/ for details.

Reply:
I've suggested https://github.com/caddyserver/website/pull/86/files#r493102988 to maybe clarify this a bit.